src/main/java/io/aiven/kafka/auth/audit/AuditorConfig.java

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright (c) 2020 Aiven, Helsinki, Finland. https://aiven.io/
+ */
+
+package io.aiven.kafka.auth.audit;
+
+import java.util.Map;
+import java.util.Objects;
+
+import org.apache.kafka.common.config.AbstractConfig;
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.common.config.ConfigException;
+
+public class AuditorConfig extends AbstractConfig {
+
+    public static final String AUDITOR_TYPE_CLASS_CONF = "aiven.acl.authorizer.auditor.class.name";
+
+    public AuditorConfig(final Map<?, ?> originals) {
+        super(configDef(), originals);
+        validate();
+    }
+
+    public static ConfigDef configDef() {
+        return new ConfigDef()
+            .define(
+                AUDITOR_TYPE_CLASS_CONF,
+                ConfigDef.Type.CLASS,
+                NoAuditor.class,
+                new ConfigDef.Validator() {
+                    @Override
+                    public void ensureValid(final String name, final Object value) {
+                        if (Objects.nonNull(value)) {
+                            assert value instanceof Class;
+                            final Class<?> clazz = (Class<?>) value;
+                            if (!Auditor.class.isAssignableFrom(clazz)) {
+                                throw new ConfigException(
+                                    name, clazz.getName(),
+                                    String.format(
+                                        "wrong auditor's fully qualified class name. Supported values are: %s",
+                                        String.join(
+                                            ", ",
+                                            UserActivityAuditor.class.getName(),
+                                            UserOperationsActivityAuditor.class.getName()
+                                        )
+                                    )
+                                );
+                            }
+                        }
+                    }
+                },
```

Comment:
We can omit this validator, the class validation is done by `getConfiguredInstance`.